ava-labs/core-mobile
# CP-8715 Pin entry feels laggy

## Description

**Ticket: [CP-8715]** 

There was delay between pressing pin pad and coloring pin dots.
Introduced 3 changes:
- bigger `pin pad` buttons so users don't hit between them where nothing happens
- we register `touch down` event instead `click` event which makes system initiate response faster to user's touch
- improved performance of rendering `pin dots` by creating dedicated view instead of generating it with `for loop` and removed couple hooks which add up to delay

## Screenshots/Videos

### Pin enter on iOS
https://github.com/ava-labs/avalanche-wallet-apps/assets/6573904/acc5e7d6-bc12-42a9-bc14-46f30e52035e

### Pin enter on Android
https://github.com/ava-labs/avalanche-wallet-apps/assets/6573904/e3269004-8c47-4838-8e79-6ddd60cb47ff


## Testing
- login pin screen should work and look as expected, just faster ⚡ 
- change pin screen also

## Checklist

Please check all that apply (if applicable)


[CP-8715]: https://ava-labs.atlassian.net/browse/CP-8715?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
